src/components/atoms/Loading/Loading.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import './Loading.css'
+
+const Loading = () => {
+  return (
+    <div className='loading-container'>
+      <div className='loading-spinner'></div>
```

Comment:
```suggestion
      <div className='loading-spinner' />
```

태그 안의 text가 없다는 걸 명시해주기 위해서 그냥 바로 닫는 것도 괜찮겠네요.
굳이 꼭 제 의견을 따를 필요는 없어요.